ioctl/cmd/action/actionhash.go

```diff
@@ -230,7 +230,26 @@ func Match(in string, matchType string) string {
 			return "(Failure)"
 		} else if in == "1" {
 			return "(Success)"
+		} else if in == "100" {
+			return "(Failure : Unknown)"
+		} else if in == "101" {
+			return "(Failure : Execution out of gas)"
+		} else if in == "102" {
+			return "(Failure : Deployment out of gas - not enough gas to store code)"
+		} else if in == "103" {
+			return "(Failure : Max call depth exceeded)"
+		} else if in == "104" {
+			return "(Failure : Contract address collision)"
+		} else if in == "105" {
+			return "(Failure : No compatible interpreter)"
+		} else if in == "106" {
+			return "(Failure : Execution reverted)"
+		} else if in == "107" {
+			return "(Failure : Max code size exceeded)"
+		} else if in == "108" {
+			return "(Failure : Write protection)"
```

Comment:
is it possible to have an error code not equal to all these here?
in that case, add an else {} clause

Reply:
Actually, if the error type is not equal to known error, default code is 100(unknown). So, I think it isn't possible to have other than these code (and at the end if it's returning ""empty just in case)

Reply:
BTW, how about using switch?

Reply:
Oh ok !!!